### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vinitawanere**

### CODE REVIEW

**Review by koustubh25 — CHANGES_REQUESTED** [review]
After checking the change request, you need to manually dismiss this review to get it merged

**Review by koustubh25 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # gha-test
+
+random shit
```
Comment (koustubh25) [inline README.md:3]: sdf

**Review by koustubh25 — COMMENTED** [review]
After checking the change request, you need to manually dismiss this review to get it merged

**Review by koustubh25 — APPROVED** [review]
After checking the change request, you need to manually dismiss this review to get it merged

File: README.md (hunk 1 above)
Comment (koustubh25) [inline README.md:3]: sdfaf

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
 # gha-test
```
Comment (koustubh25) [inline README.md:1]: random

**Review by koustubh25 — COMMENTED** [review]
After checking the change request, you need to manually dismiss this review to get it merged

**Review by koustubh25 — COMMENTED** [review]
After checking the change request, you need to manually dismiss this review to get it merged

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: After checking the change request, you need to manually dismiss this review to get it merged

**Review by koustubh25 — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: After checking the change request, you need to manually dismiss this review to get it merged

**Review by koustubh25 — DISMISSED** [review]

**Review by koustubh25 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: This is more of a safety check. After confirming the change request, you need to manually dismiss this review or resolve the conversation to get it merged

**Review by koustubh25 — DISMISSED** [review]

**Review by koustubh25 — APPROVED** [review]

**Review by koustubh25 — DISMISSED** [review]
asfdasdf

**Review by koustubh25 — APPROVED** [review]

**Review by koustubh25 — CHANGES_REQUESTED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review or resolve the conversation to get it merged

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get it merged

**Review by koustubh25 — CHANGES_REQUESTED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get it merged

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get it merged

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

**Review by koustubh25 — CHANGES_REQUESTED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

**Review by koustubh25 — APPROVED** [review]

**Review by koustubh25 — COMMENTED** [review]
a

**Review by koustubh25 — APPROVED** [review]

**Review by koustubh25 — CHANGES_REQUESTED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: something

**Review by koustubh25 — CHANGES_REQUESTED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

**Review by koustubh25 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: something

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: something

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: something

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: something

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

**Review by koustubh25 — CHANGES_REQUESTED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: something

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: something

**Review by koustubh25 — CHANGES_REQUESTED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

File: README.md (hunk 2 above)
Comment (koustubh25) [inline README.md:1]: something

**Review by koustubh25 — DISMISSED** [review]
This is more of a safety check. After confirming the change request, you need to manually dismiss this review to get the PR merged

**Review by vinitawanere — COMMENTED** [review]
j

**Review by vinitawanere — COMMENTED** [review]
kk

**Pull request closed without merging**